rootconf/default/bin/onie-self-update

```diff
@@ -79,8 +80,12 @@ cli_static_update_url=$1
     exit 1
 }
 
+shift 1
+cli_static_update_parms="$@"
+
 onie_disco=
 onie_disco=$(add_parm "$onie_disco" onie_cli_static_update_url $cli_static_update_url)
+onie_disco=$(add_parm "$onie_disco" onie_cli_static_update_parms "$cli_static_update_parms")
 
```

Comment:
I have updated the commit:

7d5352a allow onie-nos-install and onie-self-update to pass parameters to installer and updater